cli/dstack/cli/updates.py

```diff
@@ -0,0 +1,74 @@
+import os
+import uuid
+from datetime import datetime
+from pathlib import Path
+from typing import Optional
+
+import requests
+from packaging import version as pkg_version
+
+from dstack import version
+
+
+def anonymous_installation_id() -> str:
+    ai_id_path = Path.home() / ".cache" / ".dstack" / "anonymous_installation_id"
+    if not ai_id_path.exists():
+        ai_id_path.parent.mkdir(parents=True, exist_ok=True)
+        with open(ai_id_path, "w") as f:
+            ai_id = uuid.uuid4().hex
+            f.write(ai_id)
+    else:
+        with open(ai_id_path) as f:
+            ai_id = f.readline().strip()
+    return ai_id
+
+
+def get_latest_version() -> Optional[str]:
+    latest_version_url = (
+        f"https://get.dstack.ai"
+        f"/{'cli' if version.__is_release__ else 'stgn-cli'}"
+        f"/latest-version"
+        f"?anonymous_installation_id={anonymous_installation_id()}"
+    )
+    try:
+        return requests.get(latest_version_url).text.strip()
+    except Exception:
+        return None
+
+
+def _check_for_updates():
```

Comment:
The name starting with an underscore indicates that the name is private to a module. So `_check_for_updates()` should be `check_for_updates()` and other function names should start with an underscore (e.g. vice versa).